.gitlab-ci.yml

```diff
@@ -1033,6 +1047,94 @@ test_full_integration:
     reports:
       junit: results_full_integration.xml
 
+# Publish acceptance test coverage into codecov
+.template_publish_acceptance_coverage:
+  stage: publish
+  image: alpine
+  dependencies:
+    - init_workspace
+  before_script:
+    - apk add --no-cache bash curl findutils git
+    - tar xf ${CI_PROJECT_DIR}/workspace.tar.gz ./go/src/github.com/mendersoftware/mender
+    - mv go/src/github.com/mendersoftware/mender ${CI_PROJECT_DIR}/mender
+  script:
+    - cd ${CI_PROJECT_DIR}/mender
+    - if [ -d ${CI_PROJECT_DIR}/acceptance-tests-coverage ]; then
+        bash -c "bash <(curl -s https://codecov.io/bash) -Z -F acceptance -s ${CI_PROJECT_DIR}/acceptance-tests-coverage";
```

Comment:
Is the mere location enough for it to understand that this should go to the mender repository?

Reply:
If I remember correctly, it is the token which decides which repo to upload to?

Reply:
Yes. The script will figure out that this directory is a git repo, and report to codecov accordingly (it requires also a token in the environment)

Reply:
> If I remember correctly, it is the token which decides which repo to upload to?

I don't think so. As I said, the script needs to be executed in the git repo that the coverage data belongs to.

Reply:
Aha!